less/dropdowns.less

```diff
@@ -5,21 +5,21 @@
 
 // Dropdown arrow/caret
 .caret {
-    vertical-align: middle;
-    display: inline-block;
-    font-family: 'Glyphicons Halflings';
-    font-size: 14px;
-    font-style: normal;
-    font-weight: 300;
-    line-height: 1;
-    -webkit-font-smoothing: antialiased;
-    -moz-osx-font-smoothing: grayscale;
-    margin-left: 1px;
-    text-align: center;
-    width: 16px;
-    transition: transform .15s linear;
+  vertical-align: middle;
+  display: inline-block;
+  font-family: 'Glyphicons Halflings';
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: 1;
+  -webkit-font-smoothing: antialiased;
```

Comment:
Could we do `font-smooth` instead and autoprefix this?

https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth

Reply:
Not instead, but can add it.